### PR TITLE
Adding support for force flag with GH-Pages

### PR DESCRIPTION
**Pull request by safaci2000**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -78,6 +78,11 @@ func main() {
 			Usage:  "git clone path",
 			EnvVar: "PLUGIN_WORKSPACE,DRONE_WORKSPACE",
 		},
+		cli.StringFlag{
+			Name:   "force-push",
+			Usage:  "git force push",
+			EnvVar: "PLUGIN_REMOTE_URL,DRONE_REMOTE_URL",
```
Comment (safaci2000) [inline main.go:84]: I wasn't sure what to set this to, just copied a previous value.  This is my first attempt at contributing to drone plugin.

File: .drone.yml [hunk 2]
```diff
@@ -114,6 +114,26 @@ steps:
       exclude:
       - pull_request
 
+- name: publish-force
```
Comment (donny-dont) [inline .drone.yml:117]: Remove the change to the `.drone.yml` and I'll happily merge this

**Review by donny-dont — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 2 above)
Comment (safaci2000) [reply .drone.yml:117]: Done

File: main.go (hunk 1 above)
Comment (donny-dont) [reply main.go:84]: Oh sorry didn't see this part.

The environment variable should be PLUGIN_FORCE_PUSH

Comment (safaci2000) [reply main.go]: Done.

**Comment by safaci2000** [comment]
Should be good to go, let me know if there's anything else a miss.

**Review by donny-dont — APPROVED** [review]

**Comment by donny-dont** [comment]
Thanks!

**Comment by safaci2000** [comment]
Now, here's my next question. Is this immediately available? or do you need to do a release?

**Comment by donny-dont** [comment]
It's tagged as :latest so available right now. See https://hub.docker.com/r/plugins/gh-pages/

File: main.go [hunk 3]
```diff
@@ -78,6 +78,11 @@ func main() {
 			Usage:  "git clone path",
 			EnvVar: "PLUGIN_WORKSPACE,DRONE_WORKSPACE",
 		},
+		cli.BoolTFlag{
```
Comment (tboerger) [inline main.go:81]: This is bad, this is a breaking change as BoolTFlag is true by default, it had been false before.

Comment (donny-dont) [reply main.go:81]: Shit missed that. Can you just modify the file and correct it in the main branch?

Not in front of a computer

**Pull request merged**